package.json

```diff
@@ -22,7 +22,7 @@
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "^10.0.0",
     "babel-jest": "^23.6.0",
-    "bs-platform": "^5.0.0",
+    "bs-platform": "^5.0.3",
```

Comment:
Please update this deps in example/reason-react-app & in packages/bs-platform too, for consitency :)